pac4j-saml/src/main/java/org/pac4j/saml/logout/impl/SAML2LogoutRequestBuilder.java

```diff
@@ -35,8 +38,9 @@
      *
      * @param bindingType the binding type
      */
-    public SAML2LogoutRequestBuilder(final String bindingType) {
-        this.bindingType = bindingType;
+    public SAML2LogoutRequestBuilder(final SAML2Configuration cfg) {
```

Comment:
It might be good to add the configuration object as a field and directly use that now, rather than individual fields that are basically duplicates.

Reply:
@mmoayyed @minfrin Notice that this should go into a 3.9.x branch but I'll handle that after the merge...